influxql/parser_test.go

```diff
@@ -293,6 +293,19 @@ func TestParser_ParseStatement(t *testing.T) {
 			},
 		},
 
+		// SHOW TAG VALUES WITH KEY = "..."
+		{
+			s: `SHOW TAG VALUES WITH KEY = "host" WHERE region = 'uswest'`,
+			stmt: &influxql.ShowTagValuesStatement{
+				TagKeys: []string{`"host"`},
```

Comment:
We may be able to refactor the scanner / parser code to get rid of this.  I'd have to look at it.